docs/features/software-catalog/descriptor-format.md

```diff
@@ -314,6 +314,34 @@ This field is optional, and currently has no special semantics.
 Each tag must be sequences of `[a-z0-9]` separated by `-`, at most 63 characters
 in total.
 
+### `links` [optional]
```

Comment:
could you add this field to the example at the very top of the file? both in the json and yaml ones that are next to each other